static/app/views/performance/vitalDetail/utils.tsx

```diff
@@ -150,6 +151,35 @@ export const vitalToMetricsField: Record<string, TransactionMetric> = {
   [WebVital.CLS]: TransactionMetric.SENTRY_TRANSACTIONS_MEASUREMENTS_CLS,
 };
 
+export const vitalSupportedBrowsers: Partial<Record<WebVital, Browser[]>> = {
+  [WebVital.LCP]: [Browser.CHROME, Browser.EDGE, Browser.OPERA],
+  [WebVital.FID]: [
+    Browser.CHROME,
+    Browser.EDGE,
+    Browser.OPERA,
+    Browser.FIREFOX,
+    Browser.SAFARI,
+    Browser.IE,
+  ],
+  [WebVital.CLS]: [Browser.CHROME, Browser.EDGE, Browser.OPERA],
+  [WebVital.FP]: [Browser.CHROME, Browser.EDGE, Browser.OPERA],
+  [WebVital.FCP]: [
+    Browser.CHROME,
+    Browser.EDGE,
+    Browser.OPERA,
+    Browser.FIREFOX,
+    Browser.SAFARI,
+  ],
+  [WebVital.TTFB]: [
+    Browser.CHROME,
+    Browser.EDGE,
+    Browser.OPERA,
+    Browser.FIREFOX,
+    Browser.SAFARI,
+    Browser.IE,
+  ],
+};
```

Comment:
Note: I got these from [this page](https://docs.sentry.io/product/performance/web-vitals/)